src/frontend/Komponenter/Behandling/Inngangsvilkår/Aleneomsorg/AleneomsorgInfo.tsx

```diff
@@ -99,7 +119,7 @@ const AleneomsorgInfo: FC<{
                         <Søknadsgrunnlag />
                         <Normaltekst>Søkes det om barnetilsyn for barnet</Normaltekst>
                         <Normaltekst>
-                            {barnepass && barnepass.skalHaBarnepass ? 'Ja' : 'Nei'}
+                            {utledEtikett(barnMedLøpendeStønad, barnId, barnepass?.skalHaBarnepass)}
```

Comment:
Vurder å splitte opp i to forskjellige "label-verdi"-er i egen PR.

Reply:
I henhold til kommentaren i denne oppgaven: https://favro.com/organization/98c34fb974ce445eac854de0/a64c6aad9b0d61ef6c0290bd?card=Tea-8564